online_portfolio/static/portfolio/js/responsive.js

```diff
@@ -0,0 +1,55 @@
+console.log("respinsive.js")
+
+var addE = function () {
+    // console.log("add E function called!!")
```

Comment:
@HarshitaAgarwal09 maintain clean code..

Reply:
I have already removed this part in the code pushed yesterday.